deeppavlov/core/common/registry.json

```diff
@@ -162,5 +162,11 @@
   "ubuntu_v2_mt_reader": "deeppavlov.dataset_readers.ubuntu_v2_mt_reader:UbuntuV2MTReader",
   "ubuntu_v2_reader": "deeppavlov.dataset_readers.ubuntu_v2_reader:UbuntuV2Reader",
   "wiki_sqlite_vocab": "deeppavlov.vocabs.wiki_sqlite:WikiSQLiteVocab",
-  "wikitionary_100K_vocab": "deeppavlov.vocabs.typos:Wiki100KDictionary"
+  "wikitionary_100K_vocab": "deeppavlov.vocabs.typos:Wiki100KDictionary",
+  "ner_reader": "deeppavlov.dataset_readers.ner_reader:NerReader",
```

Comment:
`ner_reader` is not present in this PR.
Why didn't you just update the `conll2003_reader` to be able to do what you want?

Reply:
`conll2003_reader` is quite complex and consider some information like `doc_ids`, `doc_numer` which will not be used in my model. One more important thing is that it add "O" type to POS and tag which maybe decreases the model performance. Changing the code will affect the previous trained models.

Reply:
`ner_reader` has been added.